### PR TITLE
Create introduction.md

### PR DESCRIPTION
**Pull request by edilenedacruz**
This sassy aspiring programmer will one day have crazy skills. First I'm tackling Ruby but I truly like Code School's HTML & CSS: http://campus.codeschool.com/courses/mastering-github/level/1/section/3/pull-request

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Hi!
```
Comment (codeschool-kiddo) [inline introduction.md:1]: Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

**Comment by edilenedacruz** [comment]
So far I'm very impressed with Code School's videos and other materials. Keep up the great work!

**Comment by edilenedacruz** [comment]
Hum.. what did I do wrong? Let me try again.

**Comment by edilenedacruz** [comment]
HTML & CSS: [http://campus.codeschool.com/courses/mastering-github/level/1/section/3/pull-request](url)
